### ISSUE TITLE
Documentation should call out need for strip_beams to be false on releases

### ISSUE THREAD

**Issue by stocks29** [pylon/spect#14]
In order for `Spect` to work in a mix release, the `strip_beams` release option must be set to `false` in a project's `mix.exs` release configuration. Example:

```elixir
  def project do
    [
      app: :my_app,
      version: "0.1.0",
      elixir: "~> 1.11",
      elixirc_paths: elixirc_paths(Mix.env()),
      compilers: [:phoenix, :gettext] ++ Mix.compilers(),
      start_permanent: Mix.env() == :prod,
      aliases: aliases(),
      deps: deps(),
      default_release: :my_app,
      releases: [
        my_app: [
          # this is needed for spect to work
          strip_beams: false
        ]
      ]
    ]
  end
```

Without `strip_beams` set to false, calls to `Spect.to_spec!()` will fail with `module not found: ...` errors since the `debug_info` chunk will be missing.

**Comment by brentspell** [comment]
This should be fixed by #16.

**Comment by stocks29** [comment]
Thanks for adding this and the other fix. 

It's worth noting that strip_beams is on by default for mix releases. It must be explicitly disabled to work with releases.
